src/views/product-landing/helpers/index.ts

```diff
@@ -1,4 +1,3 @@
-export * from './validate-against-schema'
```

Comment:
Moved to `src/lib` as now used for multiple views - here in `product-landing`, and now for `home` as well.